### PR TITLE
Ecto 2 1 update

### PR DESCRIPTION
**Pull request by manukall**
This fixes deprecation warnings introduces by Ecto 2.1.

I also changed the test migration name and location, because with the old name the migration wasn't run.

### CODE REVIEW

File: config/test.exs [hunk 1]
```diff
@@ -17,3 +17,4 @@ config :guardian_db, GuardianDb.Test.Repo,
   database: "guardian_db_test",
   pool: Ecto.Adapters.SQL.Sandbox,
   priv: "test/support"
+  priv: "priv/test"
```
Comment (doomspork) [inline config/test.exs:19]: Do we need both `:priv` keys?

**Comment by doomspork** [comment]
Thank you @manukall!  I only have one question with regards to the configuration.

**Comment by manukall** [comment]
@doomspork thanks for reviewing. the double priv keys were indeed an error. the one under `priv/test` is correct, as the ecto documentation states that migrations need to be under `priv`.

**Comment by doomspork** [comment]
Thank you @manukall 👍

**Pull request merged**